hack/generator/pkg/astbuilder/assignments.go

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package astbuilder
+
+import (
+	"go/ast"
+	"go/token"
+)
+
+// SimpleAssignment performs a simple assignment like:
+// 	   <lhs> := <rhs>       // tok = token.DEFINE
+// or  <lhs> = <rhs>        // tok = token.ASSIGN
```

Comment:
minor: these two comments don't seem to line up and it's bugging me

Reply:
Tabs vs spaces. #fixed